packages/react-sdk/package.json

```diff
@@ -73,7 +73,9 @@
   "dependencies": {
     "@xmtp/xmtp-js": "^9.0.1",
     "date-fns": "^2.30.0",
-    "react": "^18.2.0"
+    "dexie": "^3.2.3",
+    "react": "^18.2.0",
+    "react-18-blockies": "^1.0.6"
```

Comment:
Did this sneak in by accident, or did we realize it should have been there all along?

Reply:
whoops, that should've went away when decoupling the React components. i think a rebase brought it back. good catch, will remove!